docs/book/src/introduction/core_library.md

```diff
@@ -0,0 +1,44 @@
+# Core Library
+
+The Sway Core Library, like the name suggests contains core operators and logic for the primitive types of the Sway programming language. These traits and methods are an extension of the [primitive types](https://docs.fuel.network/docs/sway/basics/built_in_types/#primitive-types) `u8`, `u16`, `u32`, `u64`, `u256`, `str[]`, `str`, `bool` and , `b256` and can be used where appropriate.
+
+The latest core library documentation can be found [here](https://fuellabs.github.io/sway/master/core/). If the latest version is not compatible please refer to the appropriate tagged release.
+
+## Using the Core Library
+
+Core library functionalities do not need to be explicitly imported and will work out of the box after creating any new Sway project with [`forc new`](../forc/commands/forc_new.md). The `use` keyword is simply not required.
+
+Consider this example of using the modulo function for two like value types:
+
+```sway
+let val_1 = 10;
+let val_2 = 2;
+let result = val_1 % val_2;
+```
+
+## Core Library Prelude
+
+The prelude contains a list of operations essential to all Sway programs. The latest version of the prelude can be found [here](https://github.com/FuelLabs/sway/blob/master/sway-lib-core/src/prelude.sw).
+
+- [`core::primitives::*`](https://github.com/FuelLabs/sway/blob/master/sway-lib-core/src/primitives.sw) a module for getting `max`, `min`, `bits` and `zero` for integers.
```

Comment:
I think we should just cut this whole enumeration of modules. It's going to be instantly out of date when we update core and nobody's going to remember to maintain it.

These descriptions have a proper place: they belong as module doccomments in core files. Not here.